doc/EFFECTS_JSON.md

```diff
@@ -212,6 +212,13 @@ Because the "dur_add_perc" = 150 in the example above, the second addition adds
 a total value of 250 ticks from the two. This can also be below 100%, and should be able to even be negative, leading to
 future applications decreasing the overall time left.
 
+### Duration control
+```C++
```

Comment:
I know it's wrong in many parts of the documentation, but this should "JSON", not "C++". The code inside is JSON, not C++.